doc/openapi.json

```diff
@@ -0,0 +1,67 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Sentence aligner API"
+  },
+  "tags": [
+    {
+      "name": "sentences",
+      "description": "Operations on sentences"
+    }
+  ],
+  "schemes": [
+    "http"
+  ],
+  "paths": {
+    "/sentences": {
+      "post": {
+        "tags": [
+          "sentences"
+        ],
+        "summary": "Create a new sentence.",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The sentence object that has to be created.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Sentence"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The sentence has been created."
+          },
+          "409": {
+            "description": "The given UUID is already used by another sentence."
```

Comment:
in an other MR: could be also because you have inserted a  couple  (text,lang_id) already existing

Reply:
Issue created here for future MR https://github.com/allan-simon/sentence-aligner-rust/issues/41.